test/Feature/QuerySpec.hs

```diff
@@ -265,19 +265,8 @@ spec actualPgVersion = do
         [json|[{"id":1,"name":"Windows 7","clients":{"id":1,"name":"Microsoft"},"tasks":[{"id":1,"name":"Design w7"},{"id":2,"name":"Code w7"}]}]|]
         { matchHeaders = [matchContentTypeJson] }
 
-    it "requesting parent without specifying primary key" $
-      get "/projects?select=name,client(name)" `shouldRespondWith`
-        [json|[
```

Comment:
This test has been moved to [EmbedDisambiguationSpec](https://github.com/steve-chavez/postgrest/blob/5e7d7ea8efa8ea32641c82ed5b6ca11472a27a78/test/Feature/EmbedDisambiguationSpec.hs#L144-L148)(which github hides from the default diff). It's shown to work with the FK rename.